docs/user-guide-policy-configure-ovs-bridge.md

```diff
@@ -0,0 +1,123 @@
+# Tutorial: Create a Open vSwitch Bridge and Connect it to a Node Interface
+
+Use Node Network Configuration Policy to configure a new ovs bridge `br1` connected
+to node interface `eth1`.
+
+## Requirements
+
+Before we start, please make sure that you have your Kubernetes/OpenShift
+cluster ready. In order to do that, you can follow the guides of deployment on
+[local cluster](deployment-local-cluster.md) or your
+[arbitrary cluster](deployment-arbitrary-cluster.md).
+
+You must also be sure that [OpenVSwitch](https://www.openvswitch.org/) and [NetworkManager OpenVSwitch plugin](https://developer.gnome.org/NetworkManager/stable/nm-openvswitch.html) are installed in the nodes.
+
+Please also note that on OpenShift `openvswitch` is already installed as a daemon, so only `nm-openvswitch` is required.
+
+## Configure bridge
+
+All you have to do in order to create the bridge on all nodes across cluster is
+to apply the following policy:
+
+```yaml
+cat <<EOF | kubectl create -f -
+apiVersion: nmstate.io/v1alpha1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: br1-eth1-policy
+spec:
+  desiredState:
+    interfaces:
+      - name: br1
+        description: ovs bridge with eth1 as a port
+        type: ovs-bridge
+        state: up
+        bridge:
+          options:
+            stp: false
+          port:
+            - name: eth1
+EOF
+```
+
+By doing this though, we will be able to create an Open vSwitch bridge on the host but the host may loose connectivity since its nic is now connected to the bridge.
```

Comment:
Let's not mention loss of connectivity. That is very specific issue. You may find some inspiration here https://github.com/nmstate/nmstate.github.io/blob/master/devel/api.md#ovsbridgeport_type (although it is not the best worded description. Maybe the blog post you shared with us would work better?

Reply:
@phoracek tried to rephrase, let me know if this version sounds better.

Reply:
It does not. I would not mention connectivity at all. It is just about setting an IP. If it is not for the default interface, connectivity is only distracting.